### DISCUSSION TITLE
Brain tumor 3D segmentation

### DISCUSSION THREAD

**Discussion by anuragp2018** [Project-MONAI/tutorials#1634] [General]
hey i want to download pre train weight (.pth file) for brain tumor 3D segmentation. Is it available anywhere.

**Comment by KumoLiu** [comment]
Hi @anuragp2018, you can find the pretrained weights from the model zoo.
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/large_files.yml
Hope it helps, thanks.

**Reply by anuragp2018** [reply]
and how can i used this weights please give me any reference.

**Reply by ericspod** [reply]
This bundle can be downloaded and run with the commands described here: https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation#monai-bundle-commands You can download the bundle with its weights using the bundle download command but it's probably easier to get the zip file manually: https://github.com/Project-MONAI/model-zoo/releases/download/hosting_storage_v1/brats_mri_segmentation_v0.4.8.zip

**Reply by KumoLiu** [reply]
Hi @ericspod, just in case you're unaware, since this [PR](https://github.com/Project-MONAI/model-zoo/pull/463) the model weights under release/download will not be updated.
Thanks.

**Reply by ericspod** [reply]
I think this is the most recent version of this model anyway, but yeah in general this isn't good advice now. The command `python -m monai.bundle download brats_mri_segmentation` would pull down the most recent version if this wasn't the case.
